CHANGELOG-0.7.0.md

```diff
@@ -0,0 +1,13 @@
+# Changelog since v0.6.1
+
```

Comment:
Can you also add a Breaking Changes section, that points to the 2 changes that modified the existing interfaces?

Reply:
Done